terraform/cp4ba/README.md

```diff
@@ -19,7 +19,7 @@ The development and testing of the sandbox setup code requires the following ele
 - [Install IBM Cloud CLI](https://ibm.github.io/cloud-enterprise-examples/iac/setup-environment#install-ibm-cloud-cli)
```

Comment:
Please fix format of this file to match https://github.com/ibm-hcbt/cloud-pak-sandboxes/blob/master/terraform/cp4i/README.md

Reply:
The things listed in `outputs` don't match what is really output in `outputs.tf`.  And don't really need to output the resource group since this is an input, the user would already know this information.  

Also, we do not have any defaults for the passwords, so need to remove this information in the README.